app/src/main/java/de/danoeh/antennapod/adapter/SearchlistAdapter.java

```diff
@@ -96,6 +97,12 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 holder.subtitle.setText(result.getSubtitle());
             }
 
+            if(item.isPlayed()) {
+                ViewHelper.setAlpha(convertView, 0.5f);
+            } else {
+                ViewHelper.setAlpha(convertView, 1.0f);
```

Comment:
Just a thought: ``ViewHelper.setAlpha(convertView, item.isPlayed() ? 0.5f : 1.0f);`` might even be nicer